README.md

```diff
@@ -134,9 +134,9 @@ module.exports = {
     rules: [
       {
         test: /\.less$/,
-        use: ExtractTextPlugin.extract(
+        use: ExtractTextPlugin.extract({
           fallbackLoader: 'style-loader',
-          loaders: [
```

Comment:
This is right man!! Take a time for read little more about features [webpack 2](https://webpack.js.org/), because all examples described on README are using webpack 2 and in webpack 2, it's used `rules` array for loaders instead `loaders` array. So, with rules you can use `use` array, like recommend for webpack, for to set the loaders, or you can use `loaders` array or `loader` for a single loader.